production/operator/crds/monitoring.coreos.com_podmonitors.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.9.2
+    controller-gen.kubebuilder.io/version: v0.8.0
```

Comment:
Talk to @spartan0x117, but I'm not sure we want to use an older version to regenerate these

Reply:
I used `BUILD_IMAGE=grafana/agent-build-image:0.17.0 USE_CONTAINER=1 make generate-crds`, what I'm supposed to use?

Reply:
You don't need to specify the build image, `USE_CONTAINER=1 make generate-crds` will use the latest build image.

It looks like the references never got updated after Mischa's PR, but all of our CI and Makefile stuff should be using `grafana/agent-build-image:0.19.0` now.

Reply:
ah i see, ok!

Reply:
Hmm, looks like the build for the image may have failed? https://drone.grafana.net/grafana/agent/5144/1/2

I'll put together a followup PR for updating all of the `agent-build-image` tags

Reply:
Thanks, looked like it timed out. I restarted the job: https://drone.grafana.net/grafana/agent/5199